deployment/ansible/group_vars/all

```diff
@@ -30,6 +30,7 @@ postgis_package_version: "2.3.1+dfsg-1.pgdg14.04+1"
 # azavea.docker
 # 1.11.* to match AWS ECS
 docker_version: 1.11.*
+docker_options: "--storage-opt dm.basesize=20G"
```

Comment:
After some more investigation here, I think it is possible that we are declaring a bare volume without an explicit mapping to a location on the host file system. Mappings with an explicit location on the host file system should bypass any base volume size constraints with the storage driver.

Where do we define the volume that is running into this constraint?